services/navigation/builtin/builtin.go

```diff
@@ -575,7 +581,49 @@ func (svc *builtIn) waypointIsDeleted() bool {
 func (svc *builtIn) Obstacles(ctx context.Context, extra map[string]interface{}) ([]*spatialmath.GeoObstacle, error) {
 	svc.mu.RLock()
 	defer svc.mu.RUnlock()
-	return svc.obstacles, nil
+
+	gobs := svc.obstacles
```

Comment:
I'm not sure what a gob is supposed to be.  Can you name this to something more explicit?